include/functions.h

```diff
@@ -1172,9 +1172,9 @@ void Room_Draw(PlayState* play, Room* room, u32 flags);
 void func_80097534(PlayState* play, RoomContext* roomCtx);
 void Sample_Destroy(GameState* thisx);
 void Sample_Init(GameState* thisx);
-void Inventory_ChangeEquipment(s16 equipment, u16 value);
-u8 Inventory_DeleteEquipment(PlayState* play, s16 equipment);
-void Inventory_ChangeUpgrade(s16 upgrade, s16 value);
+void Inventory_ChangeEquipment(s16 equipmentType, u16 equipValue);
+u8 Inventory_DeleteEquippedItem(PlayState* play, s16 equipmentType);
```

Comment:
wondering why the wording change on this function
It is true that it removes from your current equips if you have it equipped, but it also removes if from your inventory.
Id say that `Inventory_DeleteEquipment` makes more sense alongside `Inventory_ChangeEquipment` as well. and having the word "item" in it is prob more confusing and makes it sound like it takes an item id

Reply:
The key thing I wanted to get across is that it will delete the currently-equipped thing of that type, which the original name didn't (and which is also not how `ChangeEquipment` works). I don't think they're analogous if only because the arguments are not the same.

Reply:
Besides change/delete, what's the difference between `Inventory_ChangeEquipment` and `Inventory_DeleteEquippedItem`? Both deal with the currently equiped item

Note your comment on `Inventory_ChangeEquipment` even says "Change currently-equipped item."
while `Inventory_DeleteEquippedItem` has "Deletes the currently-equipped equipment"
It just seems natural to use either interchangeably

Reply:
either
```suggestion
void Inventory_ChangeEquipment(s16 equipmentType, u16 equipValue);
u8 Inventory_DeleteEquipment(PlayState* play, s16 equipmentType);
```
or
```suggestion
void Inventory_ChangeEquippedItem(s16 equipmentType, u16 equipValue);
u8 Inventory_DeleteEquippedItem(PlayState* play, s16 equipmentType);
```